test/dummy/app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
Doesn't having this file in the dummy app prevent the engine from using the engine layout?

Reply:
Hmm... that's a good question. I'll do some digging on this.

Reply:
I'm trying to wrap this work up this month while I'm still on maintenance, but have not made progress on this question yet. It may come down to my Rails being rusty, but I'm going to have to leave this unanswered for the moment.